tests/server/cluster/cluster_test.go

```diff
@@ -1503,32 +1503,58 @@ func TestExternalTimestamp(t *testing.T) {
 	}
 
 	{ // case3: set external timestamp larger than global ts
-		req := &pdpb.TsoRequest{
-			Header:     testutil.NewRequestHeader(clusterID),
-			Count:      1,
-			DcLocation: tso.GlobalDCLocation,
-		}
 		tsoClient, err := grpcPDClient.Tso(ctx)
 		re.NoError(err)
 		defer tsoClient.CloseSend()
-		re.NoError(tsoClient.Send(req))
-		resp, err := tsoClient.Recv()
-		re.NoError(err)
-		globalTS := tsoutil.GenerateTS(resp.Timestamp)
-
-		req2 := &pdpb.SetExternalTimestampRequest{
-			Header:    testutil.NewRequestHeader(clusterID),
-			Timestamp: globalTS + 100,
-		}
-		_, err = grpcPDClient.SetExternalTimestamp(context.Background(), req2)
-		re.NoError(err)
+		testutil.Eventually(re, func() bool {
```

Comment:
Why use Eventually?

Reply:
L1528 tries to set am external ts large than global ts in L1526. We need to ensure the global ts of L1528 still be less than L1528.

Reply:
Maybe 10s is enough.